doc_files/.onshape_client_config.yaml

```diff
@@ -1,5 +0,0 @@
-prod_api_keys:
-  base_url: "https://cad.onshape.com"
```

Comment:
We NEED to permanently wipe these from the repo. @fletchapin can you scrape the whole repo for this?

Reply:
I mean no commit should ever have these. You can use something like https://rtyley.github.io/bfg-repo-cleaner/

Reply:
Thanks for pointing that out! I added them to the .gitignore so they can't accidentally be added in the future. I'll look into deleting the history

Reply:
Just finished cleaning the repo